client/app/bundles/course/survey/pages/ResponseShow/index.jsx

```diff
@@ -117,6 +130,18 @@ class ResponseShow extends React.Component {
     );
   }
 
+  handleUnsubmitResponse = (data) => {
+    const { dispatch, params: { responseId } } = this.props;
+    const { saveSuccess, saveFailure, unsubmitSuccess, unsubmitFailure } = translations;
+    const payload = ResponseShow.formatSurveyResponseData(data);
+    const successMessage = <FormattedMessage {...(data.unsubmit ? unsubmitSuccess : saveSuccess)} />;
+    const failureMessage = <FormattedMessage {...(data.unsubmit ? unsubmitFailure : saveFailure)} />;
```

Comment:
`handleUpdateResponse` handles both save and submit, hence the `data.submit ? ... : ...` switches. We don't need it for `renderUnsubmitButton` since it should be handling one case only.

Don't think we want to allow staff to update the other fields as well. We can construct the payload directly as 
```
const payload = { response: { unsubmit: true } }
```